package.json

```diff
@@ -14,13 +14,13 @@
     "openapi-generator",
     "@onfido/api"
   ],
-  "license": "MIT",         
```

Comment:
Need to remove the space (before the comment) from generator to avoid that happens in the future again.